### PR TITLE
Update to new Ballerina Syntax

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Update Module.md

### CODE REVIEW

**Comment by daneshk** [comment]
@anuruddhal We need to change the version to `0.9.0`. we decided to go with `x.9.0` for swan lake

File: Ballerina.lock [hunk 1]
```diff
@@ -1,4 +1,329 @@
 org_name = "ballerinax"
 version = "0.1.0"
 lockfile_version = "1.0.0"
-ballerina_version = "1.2.2"
+ballerina_version = "2.0.0-Preview1-M6"
```
Comment (daneshk) [inline Ballerina.lock:4]:
```suggestion
ballerina_version = "slp1"
```

Comment (daneshk) [reply Ballerina.lock:4]: This is an auto-generated file. once you build with swan lake previews, this will auto correct

**Comment by daneshk** [comment]
Do we need to update `README` and `Module.md` files with compatible ballerina version

**Comment by anuruddhal** [comment]
@daneshk Updated the module.md and ReadME.md.

**Pull request merged**